.circleci/config.yml

```diff
@@ -32,3 +32,27 @@ jobs:
       - store_artifacts:
             path: workflows/results
             destination: results
+  docs:
+    docker:
+      - image: circleci/python:3
+    steps:
+      - checkout
+      - run:
+          name: Build documentation
+          command: |
+            cd docs/read_the_docs
+            pip install -r requirements.txt
+            make livehtml
```

Comment:
This should just be
```suggestion
            make html
```

Reply:
Ah. Because it's not built locally. Good catch.